src/Psr/CacheItemPool/CacheItemPoolDecorator.php

```diff
@@ -196,15 +195,15 @@ public function clear()
     /**
      * {@inheritdoc}
      */
-    public function deleteItem($key)
+    public function deleteItem($key): bool
```

Comment:
Can we also add argument types so we are aligned with the interface?

Reply:
added (needed to drop "psr/cache": "^1.0  for it to work)